platforms/intellij/compose/src/jvmMain/kotlin/foundry/intellij/compose/aibot/ChatWindowUi.kt

```diff
@@ -131,13 +139,13 @@ private fun ConversationField(modifier: Modifier = Modifier, onSendMessage: (Str
     Column(Modifier.fillMaxHeight(), verticalArrangement = Arrangement.Center) {
       // button will be disabled if there is no text
       IconButton(
-        modifier = Modifier.padding(4.dp).enabled(isTextNotEmpty),
+        modifier = Modifier.padding(4.dp).enabled(isTextNotEmpty && !isLoading),
```

Comment:
`IconButton` has an `enabled` parameter we can use, any reason not to use that? That should handle internal state better too

Reply:
I did it mainly for how it looks. The enabled parameter doesn't make the button less faded, so I wanted to make it more obvious it's disabled when I set it.

Reply:
interesting. Ok let's keep this but file an issue on jewel about making the button disabled appearance reflect it, feels like a bug that it doesn't